### PR TITLE
Changelog for version 4.1.0

### PR DESCRIPTION
**Pull request by georgepsarakis**
## Description

- [x] Compose Changelog for version 4.1.0 .
- [x] Move previous version changelog to history.

### CODE REVIEW

File: Changelog [hunk 1]
```diff
@@ -5,19 +5,456 @@
...
+ Contributed by **Static**.
+
+- Upgrade build environment to trusty.
+
+ Contributed by **Omer Katz**.
+
+- Add note about RPC backend not support chords (#4120)
+
+ Contributed by **Anthony Lukach**.
+
```
Comment (auvipy) [inline Changelog]: this can be chopped

File: Changelog [hunk 2]
```diff
@@ -5,19 +5,456 @@
...
+ Contributed by **Jay McGrath**.
+
+- Fix wrong type in docstring (#4067)
+
+ Contributed by **Kirill Romanov**.
+
+- Fix task eager Django setting names (#4061)
+
+ Contributed by **Ross Patterson**.
+
```
Comment (auvipy) [inline Changelog]: is this actually worth a mention?

File: Changelog [hunk 3]
```diff
@@ -5,19 +5,456 @@
...
+- Update whatsnew-4.0 (#3909)
+
+ Contributed by **Primož Kerin**.
+
```
Comment (auvipy) [inline Changelog]: this doc fix?

File: Changelog [hunk 4]
```diff
@@ -5,19 +5,456 @@
...
+ Contributed by **Felipe**.
+
+- recover loglevel for ExecStart in systemd config (#4023)
+
+ Contributed by **Yoichi NAKAYAMA**.
+
+- Fix pytest config file name
+
+ Contributed by **Samuel Dion-Girardeau**.
+
```
Comment (auvipy) [inline Changelog]: i guess it's not needed too

File: Changelog [hunk 5]
```diff
@@ -5,19 +5,456 @@
...
+- Fix link reference in the FAQ
+
+ Contributed by **Samuel Dion-Girardeau**.
+
+- Fix minor typos in the FAQ and user guide
+
+ Contributed by **Samuel Dion-Girardeau**.
+
```
Comment (auvipy) [inline Changelog]: this type of doc fix can all be incorporated in a single note i believe

File: Changelog [hunk 6]
```diff
@@ -5,19 +5,456 @@
...
+ Contributed by **Simon Schmidt**.
+
+- Rename section to [bdist_wheel] as [wheel] is considered legacy (#4033)
+
+ Contributed by **Jon Dufresne**.
+
+- Signal before_task_publish not doing what documented (#4035)
+
+ Contributed by **Javier Domingo Cansino**.
+
```
Comment (auvipy) [inline Changelog]: this can be dropped or modified

File: Changelog (hunk 2 above)
Comment (georgepsarakis) [inline Changelog]: @auvipy I am still refining these entries.

**Comment by georgepsarakis** [comment]
@auvipy thanks for reviewing this!

File: Changelog [hunk 7]
```diff
@@ -5,19 +5,308 @@
 ================
 
 This document contains change notes for bugfix releases in
-the 4.0.x series (latentcall), please see :ref:`whatsnew-4.0` for
-an overview of what's new in Celery 4.0.
+the 4.1.x series (latentcall), please see :ref:`whatsnew-4.1` for
```
Comment (georgepsarakis) [inline Changelog]: Do the release names refer to Autechre songs/albums?

File: Changelog [hunk 8]
```diff
@@ -5,19 +5,168 @@
...
+
+- **Utils**: Fix maybe_make_aware (#3850)
+
+ Contributed by **Taylor C. Richberger**.
+
+- **Task**: Fix task ETA issues when timezone is defined in configuration (#3867)
+
+ Contributed by **George Psarakis**.
+
+- Prevent consuming queue before ready on startup (#3752)
```
Comment (georgepsarakis) [inline Changelog]: This last section here is not reviewed.

**Comment by georgepsarakis** [comment]
@auvipy @thedrow this is still not finalized but must be very close. I marked one last section as incomplete. Anyone wanting to contribute please feel free, I may not be able to dedicate more time on this :disappointed: .

File: Changelog (hunk 7 above)
Comment (thedrow) [inline Changelog]: I have no idea. You'll have to ask @ask

File: Changelog (hunk 8 above)
Comment (thedrow) [inline Changelog]: What do you mean by not reviewed?

Comment (georgepsarakis) [inline Changelog]: I haven't categorized or summarized these 3-4 entries here.

File: Changelog (hunk 7 above)
Comment (georgepsarakis) [inline Changelog]: Well comparing to previous releases it seems like it :smile: :
- Cipater
- Chiastic Slide

Comment (auvipy) [inline Changelog]: seems like yes :p

**Comment by auvipy** [comment]
I can do

**Comment by georgepsarakis** [comment]
@auvipy I am available if you want any help.

**Comment by auvipy** [comment]
you could try to complete it and ping me for review

**Comment by georgepsarakis** [comment]
@auvipy done!

File: docs/history/changelog-4.0.rst [hunk 9]
```diff
@@ -0,0 +1,231 @@
+.. _changelog:
+
+================
+ Change history
+================
+
```
Comment (thedrow) [inline docs/history/changelog-4.0.rst]: Shouldn't we update this document as well?

Comment (georgepsarakis) [inline docs/history/changelog-4.0.rst]: Thanks, updated.

Comment (thedrow) [inline docs/history/changelog-4.0.rst]: But we need to create changelog.rst as well right?
Before we merge, let's make a readthedocs build to verify that this branch produces the correct results.

Comment (georgepsarakis) [inline docs/history/changelog-4.0.rst]: @thedrow the file https://github.com/celery/celery/blob/master/Changelog at the root of the repository is what you mention. I have modified it accordingly. Unless I didn't understand your comment.

Comment (auvipy) [inline docs/history/changelog-4.0.rst]: changelog-4.1.rst ?

Comment (thedrow) [inline docs/history/changelog-4.0.rst]: https://github.com/celery/celery/blob/master/docs/changelog.rst includes Changelog.
Nevermind...

**Comment by thedrow** [comment]
LGTM!
@auvipy What do you think?

**Pull request merged**